fastlane_core/lib/fastlane_core/core_ext/cfpropertylist.rb

```diff
@@ -14,5 +14,5 @@
 [Array, Hash].each do |c|
   c.send :alias_method, :to_binary_plist, :to_plist
   c.send :remove_method, :to_plist
-  c.include Plist::Emit
+  c.module_eval "include Plist::Emit"
```

Comment:
Good work. This is, of course, entirely equivalent to:

```Ruby
class Array
  alias_method :to_binary_plist, :to_plist
  remove_method :to_plist
  include Plist::Emit
end

class Hash
  alias_method :to_binary_plist, :to_plist
  remove_method :to_plist
  include Plist::Emit
end
```

At first I had Enumerator in the list as well, so this saved a lot. We could just expand it out, which could be simpler.

Reply:
I tried expanding it out like that but it didn't work (on 2.0.0), so I left it like that. I think this is a pretty short piece of code, so it should be fine